### DISCUSSION TITLE
I've answered 2 discussions but they don't gave me the Galaxy Brain badge.

### DISCUSSION THREAD

**Discussion by carlos-dubon** [community/community#18454] [Other Feature Feedback, Questions, & Ideas] [answered]
The requirement for the Galaxy Brain badge is to **answer 2 discussions**. But I´ve already done so and no badge was given to me. 

This are the discussions I´ve answered:

- [https://github.com/saadeghi/daisyui/discussions/826](https://github.com/saadeghi/daisyui/discussions/826)
- [https://github.com/AdeleD/react-paginate/discussions/381](https://github.com/AdeleD/react-paginate/discussions/381)

Is there a **secret requirement** to get the badge?

**Comment by Zerotask** [comment]
There is no secret requirement. Either it's scheduled and you get it in a few minutes / hours or it's a bug and you should report it directly to GitHub

**Reply by carlos-dubon** [reply]
How to report it?

**Comment by ale-lit** [comment, marked as answer]
I solved this problem! )

1. You need to unmark all your answers to your questions, as they do not count towards the reward, but only hinder (https://github.com/grid-js/gridjs/discussions/993 & https://github.com/kodingdotninja/use-tailwind-breakpoint/discussions/3)
2. You need to delete 1 of your already selected answers (to end up with less than 2) 😪
3. You need to get a new choice of your answer as the best from another participant and not in your question (so that it is 2)
4. Bingo! 🎉

**Reply by Kecbm** [reply]
Interesting these steps, thanks 😁

**Reply by hantv-work** [reply]
Useful information, thanks.

**Reply by saifeddineelhanoune** [reply]
how can i also win it i replied on a various repositories but the acheivement didn't unlock

**Reply by MichaellIbarra** [reply]
thanks

**Comment by HUMBLEGROUND** [comment]
I'll help you get that badge.
Please accept my answer!

More details here:

https://github.com/Schweinepriester/github-profile-achievements

**Reply by Kecbm** [reply]
Full repository, great work 😁

**Reply by hantv-work** [reply]
Useful activities.

**Comment by Kecbm** [comment]
Good night guys, I'm also trying to win the badge 😁

**Comment by tungnguyensnk** [comment]
good night

**Reply by MichaellIbarra** [reply]
good night

**Comment by nawam727** [comment]
To earn the "Galaxy Brain" badge on GitHub Discussions, you need to provide an answer that receives a significant number of reactions from the community. GitHub Discussions allows users to react to posts with various emojis, such as 👍, ❤️, 🚀, and others. To increase your chances of getting the "Galaxy Brain" badge, you should aim to provide helpful and insightful answers that resonate with the community.

Here are some tips to increase the likelihood of receiving reactions and earning the "Galaxy Brain" badge:

1. **Be Helpful:** Ensure that your answers are genuinely helpful and contribute to resolving the original question or issue.

2. **Be Detailed:** Provide detailed and comprehensive answers that cover the problem thoroughly. Include code samples, references, and any relevant links if necessary.

3. **Be Respectful:** Always maintain a respectful and friendly tone when interacting with others in the discussion.

4. **Be Timely:** Try to respond to questions as soon as possible. Timely responses are often appreciated and tend to receive more reactions.

5. **Use Formatting:** Use proper formatting, such as code blocks, lists, and headings, to make your answer easy to read and understand.

6. **Engage with the Community:** Participate in discussions actively, offer insights, and show interest in other people's questions and answers.

7. **Quality over Quantity:** Focus on quality rather than quantity. Providing a single well-thought-out answer is more likely to receive reactions than several short or incomplete responses.

Remember that the "Galaxy Brain" badge is awarded based on the reactions your answer receives, so the exact number required may vary depending on GitHub's internal algorithms and criteria.

Participating in GitHub Discussions in a positive and constructive manner not only helps you earn badges but also contributes to building a supportive and thriving community around the project you are involved in.

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @carlos-dubon. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Hazrat-Ali9** [comment]
@carlos-dubon 
How do I get a Galaxy Brain badge
Response

**Reply by carlos-dubon** [reply]
GitHub disabled badges, they can no longer be obtained.
